features/jwt_authentication.md

```diff
@@ -0,0 +1,195 @@
+
+> https://istio.io/latest/docs/tasks/security/authorization/authz-jwt/
+
+**Docs**
+
+- [x] Reference docs are published to preliminary.istio.io or the Istio wiki.
+- [x] Basic feature docs are published on preliminary.istio.io describing what the feature does, how to use it, and any caveats. 
+- [x] Release notes entries added as appropriate
+- [x] Upgrade notes entries added as appropriate
+
+**Tests**
+
+- [x] Automated integration tests cover core use cases with the feature enabled. 
+https://github.com/istio/istio/blob/1ec9f765e3b938b519e393c8a04675cfc6d0470b/tests/integration/security/jwt_test.go
+
+- [x] When disabled, the feature does not affect system stability or performance. 
+
+**API**
+
+- [x] Initial API review.
+
+**Approvals**
+
+- [x] The appropriate work group(s) have reviewed and approved promotion of the feature.
+- [x] The TOC has reviewed and approved promotion of the feature as part of the
+	roadmap for a release.
+
+---
+
+## Beta
+
+### Requirements: 
+
+**Design**
+
+- [x] Design doc describing the intention of the feature, how it will be
+	implemented, and any thoughts on how to test the feature has been approved by
+	relevant work group leads
+- [x] Feature coverage and test plans written and approved.
+
+**Docs** 
+
+- [x] Documentation on istio.io includes performance expectations; may have caveats. 
+- [x] Documentation on istio.io includes samples/tutorials. 
+- [x] Documentation on istio.io includes appropriate glossary entries. 
+- [x] All new documentation containing user actions includes istio.io tests.
+- [x] Release notes have been added. 
+- [x] Upgrade notes have been added. 
+
+**Tests**
+
+- [x] Integration tests cover feature edge cases
+- [x] End-to-end tests cover samples/tutorials
+- [x] Fixed issues have tests to prevent regressions
+- [x] Stability/stress test suite includes coverage for the feature.
+
+**Performance**
+
+- [x] Feature coverage and test plans written and approved 
+- [x] Tests exist with the feature enabled that can be integrated with our automated performance testing.
+
+**API**
+
+- [x] TOC has reviewed the API and determined it to be complete. 
+https://github.com/istio/api/blob/master/security/v1beta1/jwt.proto
+
+**Tooling**
+
+- [x] Any necessary tooling to use/debug the feature has been implemented and is complete. 
+
+**Bugs**
+
+- [x] Feature has no known major issues.
+
+**Approvals**
+
+- [x] The appropriate work group(s) have reviewed and approved promotion of the feature.
+- [x] The supportability review panel has reviewed promotion of the feature.  
+- [x] The TOC has reviewed and approved promotion of the feature as part of the
+	road map for a release.
+
+---
+
+## Stable
+
+### Requirements: 
+
+**Tests**
+
+- [x] Test all the possible fields from CRD to imporve the test coverage. 
+- [x] Fix issues in multicluster scenarios.
+
+**Performance**
```

Comment:
Consider adding a comment/plan for the Performance section.

Reply:
I think we have it already - https://istio.io/latest/blog/2020/large-scale-security-policy-performance-tests/, https://github.com/istio/tools/tree/master/perf/stability/control-plane. I don't think we need anything else for this. Will update the file with completed status

Reply:
I think this is fine to mark complete.